addons/mail/static/src/components/notification_list/notification_list_notification_group_tests.js

```diff
@@ -419,28 +419,27 @@ QUnit.test('different mail.channel are not grouped', async function (assert) {
     );
 });
 
-QUnit.test('multiple grouped notifications by document model, sorted by date desc', async function (assert) {
+QUnit.test('multiple grouped notifications by document model, sorted by sequence', async function (assert) {
     assert.expect(9);
 
     this.data['mail.message'].records.push(
         // first message that is expected to have a failure
         {
-            date: moment.utc().format("YYYY-MM-DD HH:mm:ss"), // random date
+            // random date, later than first message
+            date: moment.utc().add(1, 'days').format("YYYY-MM-DD HH:mm:ss"),
             id: 11, // random unique id, will be used to link failure to message
             message_type: 'email', // message must be email (goal of the test)
-            model: 'res.partner', // different model from second message
-            res_id: 31, // random unique id, useful to link failure to message
-            res_model_name: "Partner", // random related model name
+            model: 'res.company', // different model from first message
+            res_id: 32, // random unique id, useful to link failure to message
+            res_model_name: "Company", // random related model name
```

Comment:
No need to change model/model name.

Also why the need to change res_id?

Shouldn't it use id of message, and not id of relaterd record?

Reply:
The diff is unclear on this one.
I just moved the object below at the top, witch created this ugly, but "true" diff.
I did it to respect the id and the company/partner order of the test.

Reply:
I'm still not sure to understand the issue with this test, but I guess I'll just run the original one with the PR and see what I get.